tests/projects/ephemeral/deployment/Dockerfile-mpl

```diff
@@ -0,0 +1,8 @@
+# syntax=docker/dockerfile:1.4
+FROM python:3.9-alpine as installer
+COPY --link tests/projects/job/ /tests/projects/job/
+WORKDIR /tests/projects/job
+
+FROM installer as builder
+COPY tests/projects/job/src/hello-world.py helloworld.py
```

Comment:
It's a bit strange to me that stuff from job is pulled in here.

Reply:
Fair, you suggest we have exactly the same `src` folder in this project as well? Or do you have an entirely different idea on how to check the container is running?

Reply:
Maybe you can just do a `printenv` in the container to check if the env vars are there